main.go

```diff
@@ -369,6 +371,14 @@ func getManager(
 		os.Exit(1)
 	}
 
+	discoveryClient := discovery.NewDiscoveryClientForConfigOrDie(managedCfg)
+
+	serverVersion, err := discoveryClient.ServerVersion()
+	if err != nil {
```

Comment:
I took a quick look and I think this is true... this won't do an api-resources refresh or whatever, that could take a long time, right?